app/src/main/java/com/otaman/weather/networking/CurrentWeatherNetworkMapper.kt

```diff
@@ -11,4 +25,30 @@ fun CurrentWeatherResponse.toCurrentWeather(): CurrentWeather {
         conditionIcon = "https:${current.condition.iconCondition}",
         conditionText = current.condition.textCondition
     )
+}
+
+@RequiresApi(Build.VERSION_CODES.O)
```

Comment:
This is not a good idea. This means that this function won't work on versions prior to Android 8